src/main/app/src/components/Production.js

```diff
@@ -14,48 +15,84 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+const getMaterials = async () => {
+  const api = '/api/materials/';
+  const got = await fetch(api);
+  const json = await got.json();
+
+  if (!json.materials) return [];
+
+  return json.materials.map(materials => {
+    materials.allMaterials = {
+      onClick: () => alert(`Test: ${materials.material}`),
```

Comment:
From demo 2, TA said we should try to use non-default alerts. i.e. material-ui dialog boxes instead of the default `alert` api.

So for this I would either just have no alert, or replace with a MUI dialog: <https://material-ui.com/components/dialogs/>